### DISCUSSION TITLE
New BattleSnake Update

### DISCUSSION THREAD

**Discussion by darkin424** [Alphasians/Support#83] [Q&A] [answered]
Is there any resource for the BattleSnake? 
Visited a discussion but not much usefull since BattleSnake is updated.

**Comment by Sonichigo** [comment, marked as answer]
Hey @darkin424 , yeah BattleSnake is updated. But that's in respect to the battleground not in snakes capacity. I'll update the new resources soon, no worries.

**Comment by darkin424** [comment]
Okk
